docs/source/setup/gee.rst

```diff
@@ -1,174 +1,178 @@
+To get the best of SEPAL, we highly suggest that all users open their own GEE account and link it to SEPAL. 
+
+In this section of SEPAL documentation, we will guide you through the full process of creating a GEE account and linking it to SEPAL. 
 
 Create a GEE account
 --------------------
 
 Sign-up
 ^^^^^^^
 
-Signing up for Google Earth Engine is required in order to properly export images and data products from SEPAL as GEE assets or SEPAL applications using GEE. You will need to have a Google Account in order to sign up. If you don't have one already, you can set one up here: https://accounts.google.com/servicelogin. 
+Signing up for GEE is required to properly export images and data products from SEPAL as GEE assets or SEPAL applications using GEE. 
+
+You will need to have a Google account to sign up. If you don't have one already, you can set one up here: https://accounts.google.com/servicelogin. 
 
 To request a GEE account, please visit https://earthengine.google.com/new_signup/.
 
 .. image:: ../_images/setup/register/gee_landing.png
-   :alt: Request access to google earth engine
+   :alt: Request access to google earth engine.
    :align: center
 
-Once you have a Google Earth Engine account, you can access GEE here: https://code.earthengine.google.com/.
+Once you have a GEE account, you can access the platform here: https://code.earthengine.google.com/.
 
 .. image:: ../_images/setup/register/gee_code.png
-   :alt: gee code editor
+   :alt: GEE code editor
    :align: center
 
 .. tip::
 
-    On your first connection to the code editor it is strongly advised to follow the small tour proposed by the interface 
+    When you first connect to the code editor, it is strongly advised to take the short tour provided in the interface. 
 
     .. image:: ../_images/setup/gee/editor_tour.png
-        :alt: gee code editor tour
+        :alt: GEE code editor tour
         :align: center
 
-.. warning::
-
-    If you experience trouble white listing your Google Account to GEE please let the SEPAL team know, we may be able to help you
+.. tip::
 
-Initialize the home folder
+    If you experience trouble while linking your Google account to GEE, we encourage you to contact the SEPAL team for support.
+    
+Initialize the **Home** folder
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To use your Google earth engine account in SEPAL you need to properly set up the **home** folder. The home folder is where all your **assets** (vectors, rasters, collection, mosaics, classifications.. etc) will be exported. If this folder is missing then, SEPAL will have nowhere to point to and every export request will raise an error.
+To use your GEE account in SEPAL, you need to  set up the **Home** folder, where all your assets (vectors, rasters, collections, mosaics, classifications, etc.) will be exported. Failing to set up this folder will prevent you from successfully executing export requests.
```

Comment:
here I made a mistake, "vector" and "raster" are type of files that are normally known by GIS users. "collection" is a specific type of file in GEE. "mosaics" and "classification" are SEPAL recipes. In short I'm mixing different things at the same place. My guess is that instead we should focus on the file types in GEE only what do you think ?